src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/ModelMetadata.cs

```diff
@@ -2,7 +2,9 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Collections;
 using System.Collections.Generic;
+using System.Reflection;
```

Comment:
Yup - the intent was to trim down Common so it's easier to identify things that are truly common (and perhaps could go end up in the Common repo).